pkg/chartutil/values_test.go

```diff
@@ -25,6 +25,22 @@ import (
 	"helm.sh/helm/v3/pkg/chart"
 )
 
+func TestAsMap(t *testing.T) {
```

Comment:
One thing from this PR we do not have is a function like this to test `AsMap()`. This could still be added if you want to update this PR. The new file location is https://github.com/helm/helm/blob/main/pkg/chart/v2/util/values_test.go.